federated-aws-rp.yaml

```diff
@@ -149,23 +117,14 @@ Resources:
       # preventing this resource from creating
       LogGroupName: !Join [ '/', ['/aws/lambda', !Ref 'AwsFederatedRpFunction' ] ]
       RetentionInDays: 14
-  AwsFederatedRpDomainName:
-    Type: AWS::ApiGateway::DomainName
-    Condition: UseCustomDomainName
-    Properties:
-      RegionalCertificateArn: !Ref CertificateArn
-      DomainName: !Ref CustomDomainName
-      EndpointConfiguration:
-        Types:
-          - REGIONAL
   AwsFederatedRpRoute53RecordSet:
     Type: AWS::Route53::RecordSet
     Condition: UseCustomDomainName
     Properties:
       AliasTarget:
-        DNSName: !GetAtt AwsFederatedRpDomainName.RegionalDomainName
-        HostedZoneId: !GetAtt AwsFederatedRpDomainName.RegionalHostedZoneId
-      Comment: Bind the custom domain name to the AwsFederatedRp API Gateway
+        DNSName: !GetAtt AwsFederatedRpCloudFrontDistribution.DomainName
+        HostedZoneId: Z2FDTNDATAQYW2  # https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-route53-aliastarget-1.html
```

Comment:
Maybe break this out to a default param for reusability.

Reply:
@andrewkrug This is actually a global hard coded value (see that link in the comment) and should always be the same for all AWS customers